### PR TITLE
webui: Prompt user for updating webui if an update is available

### PR DESCRIPTION
**Pull request by negative0**
#### What is the purpose of this change?

This change will check if there is an available update to the webui and notify the user about it.

#### Was the change discussed in an issue or in the forum before?

No
#### Checklist

- [x] I have read the [contribution guidelines](https://github.com/rclone/rclone/blob/master/CONTRIBUTING.md#submitting-a-pull-request).
- [x] I have added tests for all changes in this PR if appropriate.
- [x] I have added documentation for the changes if appropriate.
- [x] All commit messages are in [house style](https://github.com/rclone/rclone/blob/master/CONTRIBUTING.md#commit-messages).
- [x] I'm done, this Pull Request is ready for review :-)

### CODE REVIEW

File: fs/rc/webgui/webgui.go [hunk 1]
```diff
@@ -55,17 +55,25 @@ func CheckAndDownloadWebGUIRelease(checkUpdate bool, forceUpdate bool, fetchURL
...
 
+	// Get the latest release details
+	WebUIURL, tag, size, err := GetLatestReleaseURL(fetchURL)
+	if err != nil {
+		return errors.Wrap(err, "Error checking for web gui release update, skipping update")
+	}
+	dat, err := ioutil.ReadFile(tagPath)
+
+	// Check if current version is equal to latest release
+	tagsMatch := err == nil && string(dat) == tag
```
Comment (ncw) [inline fs/rc/webgui/webgui.go:66]: I think the error handling is wrong here...

If err != nil then you'll declare that the tags don't match and you'll print the error

File: fs/rc/webgui/webgui.go [hunk 2]
```diff
@@ -55,17 +55,25 @@ func CheckAndDownloadWebGUIRelease(checkUpdate bool, forceUpdate bool, fetchURL
...
+
+	if !tagsMatch {
+		fs.Infof(nil, "A release (%s) for gui is present at %s. Use --rc-web-gui-update to update", tag, WebUIURL)
```
Comment (ncw) [inline fs/rc/webgui/webgui.go:69]: I think this should probably show your version tag and the new version tag so the user can gauge whether it is worth doing

**Review by ncw — COMMENTED** [review]
See inline :-)

File: fs/rc/webgui/webgui.go (hunk 2 above)
Comment (negative0) [reply fs/rc/webgui/webgui.go:69]: On it

**Comment by ncw** [comment]
Looks great - thank you :-)

I'll merge it now.

**Pull request merged**